src/Batch/FileStaging/Tests/BatchClientIntegrationTests/BatchClientIntegrationTests.csproj

```diff
@@ -0,0 +1,181 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I feel like this file should be just IntegrationTests, or maybe FileStagingIntegrationTests, not BatchClientIntegrationTests